### PR TITLE
BaSD-MS-87: App Authentication

### PR DESCRIPTION
**Pull request by NicoCagnina**
## App Authentication
Created app authentication logic to:

- Postulants
- Psychologists (view not created)
- Admins

### Functions:

- Register postulant
- Login 
    - Postulants 
    - Psychologists 
    - Admins
- Logout

Test users:

> Admin: admin@rr.com
_________________________
> Postulant: postulant@rr.com
_________________________
 Passwords: `12345678`

### CODE REVIEW

**Review by ravegliamartin — APPROVED** [review]

**Comment by Brunomoyano4** [comment]
In the login screen, if you try to enter with a not register mail, you can view the error in the image.
I think, could be changed for a simple text error, for example, "Email or Password has incorrect" 

login error

File: src/Components/Auth/Register/index.js [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+  const dispatch = useDispatch();
+  const loading = useSelector((store) => store.postulants.isLoading);
+  const history = useHistory();
+
+  const onSubmit = (formValues) => {
+    const credentials = {
+      email: formValues.email,
+      password: formValues.password
+    };
+    console.log(credentials);
```
Comment (Brunomoyano4) [inline src/Components/Auth/Register/index.js:21]: Delete console.log()

File: src/Routes/index.js [hunk 2]
```diff
@@ -11,7 +12,9 @@ const Routes = () => {
         <Switch>
           <Route path="/postulant" exact component={PostulantRoutes} />
           <Route path="/admin" component={AdminRoutes} />
-          <Redirect to="/postulant?id=61bd751a1d95a30cbccee7c2" />
+          <Route path="/auth" component={AuthRoutes} />
+          <Redirect to="/auth" />
+          {/* <Redirect to="/postulant?id=61bd751a1d95a30cbccee7c2" /> */}
```
Comment (Brunomoyano4) [inline src/Routes/index.js:17]: don´t forget to delete test id postulant

**Review by Brunomoyano4 — CHANGES_REQUESTED** [review]
Looks good, only change 2 details

**Review by Brunomoyano4 — APPROVED** [review]
Very good job!

**Pull request merged**